flocker/control/schema/types.yml

```diff
@@ -1,16 +1,19 @@
 $schema: http://json-schema.org/draft-04/schema#
 id: http://api.clusterhq.com/v1/types.json
 definitions:
+  uuid:
```

Comment:
I suspect supporting the specific case of allOf with only a single item wouldn't be too hard.